js/renovation/ui/scroll_view.tsx

```diff
@@ -4,34 +4,173 @@ import {
   JSXComponent,
   Slot,
   OneWay,
+  Method,
+  Ref,
 } from 'devextreme-generator/component_declaration/common';
+import { isNumeric } from '../../core/utils/type';
+import { Widget } from './common/widget';
+import config from '../../core/config';
 
-export const viewFunction = (viewModel: ScrollView) => (
-  <div className={viewModel.cssClasses}>
+const DIRECTION_VERTICAL = 'vertical';
+const DIRECTION_HORIZONTAL = 'horizontal';
+const DIRECTION_BOTH = 'both';
+const SCROLLABLE_CONTENT_CLASS = 'dx-scrollable-content';
+
+export interface Location {
```

Comment:
Why interface instead of type?

Reply:
@churkin  I can't use the type instead of the interface because ESLint has thrown the "Use an `interface` instead of a `type`.(@typescript-eslint/consistent-type-definitions" error.